ci/domain-broker-certs.sh

```diff
@@ -20,25 +20,26 @@ done
 nlbs=0
 ncerts=0
 cert_names=""
+cert_expirations=""
 for lb_arn in "${lb_arns[@]}"; do
   lb_listener_arns=$(aws elbv2 describe-listeners --load-balancer-arn "${lb_arn}" \
       | jq -r ".Listeners[] | select(.Port == 443) | .ListenerArn")
   for lb_listener_arn in ${lb_listener_arns}; do
     nlbs=$((nlbs + 1))
     certs_listener=$(aws elbv2 describe-listener-certificates --listener-arn "${lb_listener_arn}")
     cert_names="${cert_names}"$'\n'"$(echo ${certs_listener} | jq -r '.Certificates[] | .CertificateArn'  | awk -F/ '{ print $NF }')"
+      for cert_name in ${cert_names}; do
+        cert_metadata=$(aws iam get-server-certificate --server-certificate-name ${cert_name})
+        cert_date=$(echo "${cert_metadata}" | jq -r '.ServerCertificate | .ServerCertificateMetadata | .Expiration')
```

Comment:
Quick note here for others reviewing, in case this crosses their mind as well: this threw me off for a second because I was like "why does the `$cert_metadata` need to be `echo`ed here when we just captured it..." - it's because `jq`'s default behavior of reading from the input stream is being used, so the `echo` statement helps get things started by making its value available in the I/O stream, which `jq` then reads from.

It is possible to tell `jq` to read from a file or elsewhere, but this is the easier and more straightforward way to go.